core/src/main/java/org/opensearch/sql/calcite/utils/CalciteToolsHelper.java

```diff
@@ -3,28 +3,6 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/*
- * This file contains code from the Apache Calcite project (original license below).
- * It contains modifications, which are licensed as above:
- */
-
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to you under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
```

Comment:
todo: figure out a config to preserve these bits

Reply:
done, added an ignore rule if the content matches "(original license below)" (assuming this means it's a mixed-license file)